cumulus_library/template_sql/codeable_concept_denormalize.sql.jinja

```diff
@@ -79,7 +79,7 @@ CREATE TABLE {{ target_table }} AS (
             UNNEST(s.{{ field_alias }}.coding) AS u (coding)
         {%- if filter_priority %}
         WHERE
-            u.coding.system LIKE '{{ system }}'
+            REGEXP_LIKE(u.coding.system, '{{ syntax.wildcard_to_regex(system) }}')
```

Comment:
This instance and the one in coding are all caps due to sqlfluff enforcement picking up style from the unnest. Elsewhere it complained about it being lowercase. This is probably an issue with the athena sqlfluff dialect.